examples/java-matter-controller/java/src/com/matter/controller/Off.java

```diff
@@ -0,0 +1,37 @@
+/*
+ *   Copyright (c) 2022 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+
+package com.matter.controller;
+
+import com.matter.controller.commands.common.CredentialsIssuer;
+import com.matter.controller.commands.common.MatterCommand;
+import java.util.concurrent.atomic.AtomicLong;
+
+public final class Off extends MatterCommand {
+  private AtomicLong mNodeId = new AtomicLong();
+  private AtomicLong mFabricId = new AtomicLong();
```

Comment:
These two fields can be final (the references don't need to be mutable, just the contents)